hazelcast-jet-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/kafka/PropertiesResolver.java

```diff
@@ -49,11 +34,34 @@
     static final String VALUE_SERIALIZER = "value.serializer";
     static final String VALUE_DESERIALIZER = "value.deserializer";
 
-    static final String AVRO_SERIALIZER = "io.confluent.kafka.serializers.KafkaAvroSerializer";
-    static final String AVRO_DESERIALIZER = "io.confluent.kafka.serializers.KafkaAvroDeserializer";
+    // using strings instead of canonical names to not fail without Kafka on the classpath
```

Comment:
Do we care? It will fail sooner or later due to some `NoClassDefFoundError`. Especially the NullSerializer - it's not in kafka and we might rename/move it, the kafka serializers are unlikely to change FQN.